Wikipedia/UI-V5/WMFSearchViewController.m

```diff
@@ -41,19 +56,26 @@ - (void)setRecentSearchesHidden:(BOOL)hidingRecentSearches animated:(BOOL)animat
 
 @implementation WMFSearchViewController
 
-- (WMFArticleListTransition*)listTransition {
-    return self.resultsListController.listTransition;
++ (instancetype)searchViewControllerWithSite:(MWKSite*)site dataStore:(MWKDataStore*)dataStore {
+    WMFSearchViewController* searchVC = [self wmf_initialViewControllerFromClassStoryboard];
+    searchVC.searchSite             = site;
+    searchVC.dataStore              = dataStore;
+    searchVC.modalPresentationStyle = UIModalPresentationOverFullScreen;
+    searchVC.modalTransitionStyle   = UIModalTransitionStyleCrossDissolve;
+    return searchVC;
 }
 
-- (void)setSavedPages:(MWKSavedPageList* __nonnull)savedPages {
-    _savedPages = savedPages;
-    // resultsListController might be nil if `prepareForSegue:sender:` hasn't been called
-    self.resultsListController.savedPages = savedPages;
+- (instancetype)initWithCoder:(NSCoder*)aDecoder {
```

Comment:
oops, forgot why i originally had that there, but must've forgot to remove it when it became unnecessary